src/components/Chart.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import GenerationTypeChart from "./GenerationType";
+import React from "react";
+
+export enum DataType {
+    generationType = "https://data.dev.elexon.co.uk/bmrs/api/v1/generation/outturn/summary?from=2023-07-12&to=2023-07-12"

```

Comment:
I think this is quite a bizarre thing to be storing as an enum value, especially one called `DataType`